Sources/Testing/Events/Recorder/Event.ConsoleOutputRecorder.swift

```diff
@@ -148,17 +148,19 @@ extension Event.Symbol {
   /// - Returns: A string representation of `self` appropriate for writing to
   ///   a stream.
   fileprivate func stringValue(options: Set<Event.ConsoleOutputRecorder.Option>) -> String {
+    var useColorANSIEscapeCodes = false
```

Comment:
If you wanted to you could use an if expression here and avoid making this mutable. 

```swift
let useColorANSIEscapeCodes = if let colorBitDepth = options.colorBitDepth {
  colorBitDepth >= 4
} else {
  false
}

Reply:
Sure, but this is one line shorter. :D